rudder-agent/SPECS/rudder-agent.spec

```diff
@@ -139,6 +143,19 @@ Source1: Makefile
 AutoReq: 0
 AutoProv: 0
 
+%if "%{requires_python}" == "true" && 0%{?rhel} >= 9
```

Comment:
why do you need a variable since the knowledge of which distribution needs what is embedded here ? This is duplicate